hy/cmdline.py

```diff
@@ -111,9 +119,17 @@ def ast_callback(main_ast, expr_ast):
             return False
 
         if value is not None:
+            # Shift exisitng REPL results queue
+            for i in xrange(1, self._repl_results_queue_len):
```

Comment:
Stick to Python 3 isms (in this case, prefer `range` over `xrange`) and leave Python 2 a second class citizen.

Its harsh, and I wish we could drop Python 2 support, but I also don't think any of the maintainers are running Hy on Python 2 either.